docusaurus/docs/cloud/getting-started/usage-billing.md

```diff
@@ -44,14 +45,22 @@ Strapi Cloud offers a free 14 days trial for all new accounts, and 3 paid plans:
   - Releases refers to the maximum number of pending releases that can be created (see [Releases in User Guide](/user-docs/releases/introduction) for more information).
   - Review Workflows refers to the maximum number of workflows that can be created and used (see [Review Workflows in User Guide](/user-docs/settings/review-workflows) for more information).
   - Content History refers to the maximum numbers of days kept in history (see [Content History in User Guide](/user-docs/content-manager/working-with-content-history))
-- Cloud specific feature: Backups refers to the automatic backups of Strapi Cloud projects (see [dedicated page in Cloud documentation](/cloud/projects/settings#backups) for more information).
+- Cloud specific feature:
+  - Backups refers to the automatic backups of Strapi Cloud projects (see [Backups documentation](/cloud/projects/settings#backups) for more information on the feature).
+  - Environments refers to the number of environments included in the plan on top of the default production environment (see [Environments](/cloud/projects/settings#environments--) documentation for more information on the feature).
 :::
 
+## Environments management
+
+Environments are isolated instances of your Strapi Cloud project. All projects have a default production environment, but other additional environments can be configured for projects on a Pro or Team plan, from the *Environments* tab of a project's settings (see [Environments](/cloud/projects/settings#environments--)). There is no limit to the number of additional environments that can be configured for a Strapi Cloud project.
```

Comment:
internel tip:
fyi atm the limit is set at `99` but I agree it's maybe not required to mention it; as we could increase it if needed (I doubt it would be needed as it would mean to pay more than 99*69$ for pro which is 6831$ 😆)